crates/integration-tests/src/tests/filtering.rs

```diff
@@ -76,6 +76,17 @@ async fn filters_by_comparisons_on_elements_of_array_of_scalars() -> anyhow::Res
 #[tokio::test]
 async fn filters_by_comparisons_on_elements_of_array_of_scalars_against_variable(
 ) -> anyhow::Result<()> {
+    // Skip this test in MongoDB 5 because the example fails there. We're getting an error:
+    //
+    // > Kind: Command failed: Error code 5491300 (Location5491300): $documents' is not allowed in user requests, labels: {}
+    //
+    // This doesn't affect native queries that don't use the $documents stage.
+    if let Ok(image) = std::env::var("MONGODB_IMAGE") {
+        if image == "mongo:5" {
+            return Ok(());
+        }
+    }
+
```

Comment:
Do we still need mongodb 5 tests? We stopped supporting it. In the readme we state 6 and above. Also 5 is EOL Oct 2024.

Reply:
I think it would make sense to disable v5 testing in another PR